packages/@react-spectrum/s2/src/Tabs.tsx

```diff
@@ -10,9 +10,302 @@
  * governing permissions and limitations under the License.
  */
 
-import {Tabs as RACTabs, TabsProps} from 'react-aria-components';
+import {TabListProps as AriaTabListProps, TabPanel as AriaTabPanel, TabPanelProps as AriaTabPanelProps, TabProps as AriaTabProps, TabsProps as AriaTabsProps, Provider, Tab as RACTab, TabList as RACTabList, Tabs as RACTabs, TabListStateContext} from 'react-aria-components';
+import {Collection, DOMRef, Key, Node, Orientation} from '@react-types/shared';
+import {createContext, forwardRef, ReactNode, useCallback, useContext, useEffect, useRef, useState} from 'react';
+import {focusRing, getAllowedOverrides, UnsafeStyles, StylesPropWithHeight, StyleProps} from './style-utils' with {type: 'macro'};
+import {IconContext} from './Icon';
+import {keyframes, raw} from '../style/style-macro' with {type: 'macro'};
+import {size, style} from '../style/spectrum-theme' with {type: 'macro'};
+import {useDOMRef} from '@react-spectrum/utils';
+import {useLayoutEffect} from '@react-aria/utils';
+import {useLocale} from '@react-aria/i18n';
 
+interface TabsProps extends Omit<AriaTabsProps, 'className' | 'style' | 'children'>, UnsafeStyles {
+  /** Spectrum-defined styles, returned by the `style()` macro. */
+  styles?: StylesPropWithHeight,
+  /** The content to display in the tabs. */
+  children?: ReactNode,
+  density?: 'compact' | 'regular'
+}
+
+interface TabProps extends Omit<AriaTabProps, 'children' | 'style' | 'className'> {
+  children?: ReactNode
+}
+
+interface TabListProps<T> extends Omit<AriaTabListProps<T>, 'children' | 'style' | 'className'>{
+  children?: ReactNode
+}
+
+interface TabPanelProps extends Omit<AriaTabPanelProps, 'children' | 'style' | 'className'>{
+  children?: ReactNode
+}
+
+export function TabPanel(props: TabPanelProps) {
+  return (
+    <AriaTabPanel {...props} className={style({marginTop: 4, color: 'neutral'})} />
+  );
+}
+
+const tab = style({
+  ...focusRing(),
+  display: 'inline-flex',
```

Comment:
why inline? in our older implementation they were block elements which were typically flex children